web/src/components/Table/FilterPopup.tsx

```diff
@@ -0,0 +1,205 @@
+//@ts-nocheck
+
+import React, { useEffect, useImperativeHandle, useRef, useState, Ref } from 'react';
+import vegaEmbed from 'vega-embed'
+
+interface FilterPopupProps {
+  column: string;
+  onApply: (value: string) => void;
+  onClose: () => void;
+  anchorRef: HTMLDivElement;
+  filterType: any;
+  rows: any;
+  ref: any;
+}
+
+export interface FilterPopupHandle {
+  clearFilters: () => void;
+}
+
+let filter_boundaries: any = {};
+
+let brushFilterSpec: any = {
+  "width": 50,
+  "height": 12,
+  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+  "data": {"values":[]},
+  "mark": "tick",
+  "encoding": {
+      "tooltip": {"field": "value", "type": "quantitative"},
+      "x": {
+          "field": "value",
+          "type": "quantitative",
+          "scale": {"type": "linear", "zero": false},
+          "axis": {
+              "title": null,
+              "orient": "top",
+              "labelFontWeight": "lighter"
+          }
+      },
+      "color": {"condition": {"param": "selection", "value": "#0275d8"}, "value": "grey"}
+  },
+  "params": [{"name": "selection", "select": "interval"}],
+  "config": {"axis": {"grid": false},"background": null, "style": {"cell": {"stroke": "transparent"}}, "tick": {"thickness": 0.5, "bandSize": 10}}
+};
+
+
+function FilterPopup({ column, onApply, onClose, anchorRef, filterType, rows }: FilterPopupProps, ref: Ref<FilterPopupHandle>) {
+  const popupRef = useRef<HTMLDivElement>(null);
+  const chartRef = useRef<HTMLDivElement>(null);
+  const [tickOptions, setTickOptions] = useState<string[]>([]);
+  const [checkedOptions, setCheckedOptions] = useState<string[]>([]);
+
+  const clearFilters = () => {
+    alert('...');
```

Comment:
This is probably still there for developing/debugging?

Reply:
It is not ready yet still working on it